web-app/webpack.config.js

```diff
@@ -1,12 +1,19 @@
 const webpack = require('webpack');
 const path = require('path');
+const glob = require('glob-all')
 const config = require('sapper/config/webpack.js');
 const pkg = require('./package.json');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const PurifyCSSPlugin = require('purifycss-webpack');
+const WebpackPwaManifest = require('webpack-pwa-manifest');
 
-const mode = process.env.NODE_ENV;
+const mode = process.env.NODE_ENV || 'production';
 const dev = mode === 'development';
 
-const alias = { svelte: path.resolve('node_modules', 'svelte') };
+const alias = { 
+  svelte: path.resolve('node_modules', 'svelte'),
```

Comment:
It's actually weird that this alias is even defined, because in theory any import that doesn't start with either an alias or `.` is supposed to be evaluated relative to `node_modules`